modules/ROOT/pages/configure-teams.adoc

```diff
@@ -0,0 +1,138 @@
+
+To configure permissions for teams in the Teams feature interface:
+
+include::partial$include-nav-steps-team-maintainer.adoc[]
+
+. Click the team for which you want to configure permissions.
+. Click the *Permissions* tab.
+. In the *Permissions* section, click *Add permissions*
+. In the *Select Permissions* window, select the permissions that you want to add, and then select *Select contexts*.
+. Select the business groups and environments to which you want to apply the permissions, and then select *Review*. +
+The permissions you selected appear alongside the business groups and environments to which they apply.
++
+. Select *Apply permissions*. +
+The permissions you selected appear in the *Permissions* section for your team.
+
+== Manage Team Users
+
+If you have the Organization Administrator permission in your root organization, you can add users to existing teams using three methods:
+
+* The Teams feature interface
+* Sending an invitation to a new user via email
+* External group mapping
+
+[[invite-users-with-teams]]
+=== Add Users Via the Teams Interface
+
+include::partial$include-nav-steps-team-maintainer.adoc[]
+
+. In the *Teams* page, select the team to which you want to add a user.
+. Select *Add members*.
+. Select the name of each user you want to add to the team.
+. Select the membership status that you want to assign to each user: team member or team maintainer. + 
+A user who is added as a team maintainer receives team maintainer permissions as soon as you confirm their membership. 
++
+. Select *Add member*. +
+The user is added to the team and receives the team’s permissions.
+
+=== Add Users Via Email Invitations
+
+When you send an email to invite a new user to your organization, you can specify teams to which the user is added automatically when they accept your invitation. The user is also automatically added to your organization’s root team, known as “Everyone at <Root Organization>” by default.
+
+For more information about inviting a user to an existing team via email, see xref:users.adoc#inviting-a-user-to-anypoint-platform[Inviting a User to Anypoint Platform].
+
+=== Add Users Via External Group Mapping
+
+If you use single sign-on (SSO) using SAML, OIDC, or LDAP, you can map SSO users to teams. For more information, see xref:map-users-roles-teams.adoc[Map SSO Users to Teams].
+
+=== Manage Team Members and Maintainers
+
+When you add a user to a team, you can select whether that user is a team maintainer or a team member. You must have the Organization Administrator permission or be a team maintainer yourself to modify team members and team maintainers.
+
+You can designate a team maintainer to delegate team maintenance responsibilities to users without giving them additional permissions, or you can remove the team maintainer designation and convert users back to team members.
```

Comment:
Team maintainers are still members of the team - and still get its permissions - so this section reads funny to me. Maybe `without giving them full organization administrator permissions`?

Reply:
```suggestion
You can designate a team maintainer to delegate team maintenance responsibilities to users without giving them full organization administrator permissions, or you can remove the team maintainer designation and convert users back to team members.
```